server/src/test/java/org/opensearch/action/support/nodes/TransportClusterStatsActionTests.java

```diff
@@ -33,48 +33,12 @@
 
 public class TransportClusterStatsActionTests extends TransportNodesActionTests {
 
-    /**
-     * By default, we send discovery nodes list to each request that is sent across from the coordinator node. This
```

Comment:
Can we add the test to assert for actions that require the DiscoveryNode to be set in request ?

Reply:
this should be in TransportNodesAction